create_box/main.py

```diff
@@ -2,7 +2,21 @@
 
 
 def create_box(height, width, character):
-    pass
+    if character == '*':
+        x = ""
+        for i in range(height):
+            x = x + ('*' * width) + "\n"
+        return x
+            
+            
+    elif character == '@':
+        x = ""
+        for i in range(height):
+            x = x + ('@' * width)+ "\n"
+        return x 
```

Comment:
Ravs, the idea here is to use the "character" provided as input. You can use the same code to make it work. The `create_box` function should work for any character eg: `create_box(3, 4, 'A')`